frontend/src/components/Comment/PostComment.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./CommentAPI";
+import { postComment } from "./CommentAPI";
+import "./PostComment.scss";
+
+class PostComment extends React.Component {
+
+    comment(e) {
+        e.preventDefault();
+        postComment(e.target.comment.value, this.props.postId).then(
+            () => {
+                this.props.onCommentPosted();
+            }
+        ).catch((e) => {
+            // todo: proper error feedback
+            if (e.response) {
+                if (e.response.status === 401) {
+                    alert("You must be logged in to comment.")
+                } else if (e.response.data.errors) {
+                    alert("Your comment cannot be blank.");
```

Comment:
We could always keep it this way or just simply disregard the comment. We could alert the server was blank through a certain response number and have it only post a comment once a specific response code is given.

I don't know how easy it would be to implement, but I was thinking the "post comment" button would only be "available" to be clicked once the user had written something (and unavailable when the comment field is empty)